Tests/Cosmos.Kernel.Tests.Fat/System.IO/FileTest.cs

```diff
@@ -154,16 +154,16 @@ public static void Execute(Debugger mDebugger)
             using (var xFile2 = File.Create(@"0:\testdir\file.txt"))
             {
                 string wText = "This a test";
-                byte[] xWriteBuff = wText.GetUtf8Bytes(0, (uint)wText.Length);
+                byte[] xWriteBuff = Encoding.UTF8.GetBytes(wText);
                 xFile2.Write(xWriteBuff, 0, xWriteBuff.Length);
                 mDebugger.Send("---- Data written");
                 xFile2.Position = 0;
                 byte[] xReadBuff = new byte[xWriteBuff.Length];
                 xFile2.Read(xReadBuff, 0, xWriteBuff.Length);
-                mDebugger.Send("xWriteBuff=" + xWriteBuff.GetUtf8String(0, (uint)xWriteBuff.Length));
-                mDebugger.Send("xReadBuff =" + xReadBuff.GetUtf8String(0, (uint)xReadBuff.Length));
-                string xWriteBuffAsString = xWriteBuff.GetUtf8String(0, (uint)xWriteBuff.Length);
-                string xReadBuffAsString = xReadBuff.GetUtf8String(0, (uint)xReadBuff.Length);
+                mDebugger.Send("xWriteBuff=" + Encoding.UTF8.GetString(xWriteBuff));
+                mDebugger.Send("xReadBuff =" + Encoding.UTF8.GetString(xReadBuff));
+                string xWriteBuffAsString = Encoding.UTF8.GetString(xWriteBuff);
+                string xReadBuffAsString = Encoding.UTF8.GetString(xReadBuff);
                 mDebugger.Send("xWriteBuffAsString=" + xWriteBuffAsString);
```

Comment:
As Encoding.UTF8.GetString() is a costly operation do it only once the code could be changed in this way:

```cs
string xWriteBuffAsString = Encoding.UTF8.GetString(xWriteBuff);
string xReadBuffAsString = Encoding.UTF8.GetString(xReadBuff);
mDebugger.Send("xWriteBuff=" + xWriteBuffAsString ));
mDebugger.Send("xReadBuff =" + xReadBuffAsString );
```